packages/certs-v5/test/commands/certs/auto/enable.--wait.js

```diff
@@ -0,0 +1,218 @@
+'use strict'
```

Comment:
This should be moved into the `auto/enable.js` test file since we try to keep a 1:1 relationship between commands and tests.

Reply:
Done! Didn't know about the 1:1 relationship... TIL